### ISSUE TITLE
Integration: Specify inputs to transcation

### ISSUE THREAD

**Issue by johng** [btcsuite/btcd#1732]
Currently it's not possible to specify inputs when constructing a transaction using the in memory wallet

**Comment by Roasbeef** [comment]
That's correct, then `CreateTransaction` method can extended to accept inputs (or just make a new version really), then skip out on the call to `fundTx`, and use those inputs directly.
